renku/cli/run.py

```diff
@@ -181,8 +181,8 @@
 Exit codes
 ~~~~~~~~~~
 
-All Unix commands return a number between 0 and 255 which is called
-"exit code". In case other numbers are returned, they are treaded module 256
+All Unix commands return a number between 0 and 255 which is called an
+"exit code". In case other numbers are returned, they are treated modulo 256
```

Comment:
:cry:

Reply:
I honestly didn't know how to rephrase that...